test/extended/testdata/jenkins-plugin/build-job.xml

```diff
@@ -27,6 +27,22 @@
       <authToken></authToken>
       <followLog>true</followLog>
       <checkForTriggeredDeployments>true</checkForTriggeredDeployments>
+
```

Comment:
i had originally envisioned a vanilla build test and and then a test with the build env vars .... i'm assuming you are worried about the time already being spent and wanted to consolidate (of course correct my assumption as needed) .... i could be swayed, but i'm inclined to have separate tests, one for vanilla build, one for build with env vars